app/models/game.rb

```diff
@@ -36,6 +36,17 @@ def check?
     kings_in_check?('white') || kings_in_check?('black')
   end
 
+  def forfeit(current_user)
+    if current_user.id == white_player_id
+      white_player_id.update_attributes(white_player_id.losses: white_player_id.losses + 1)
```

Comment:
when using `white_player.increment(:wins, by = 1)` I get the NoMethodError `undefined method `increment' for nil:NilClass` - is my syntax wrong?